client/python/openlineage/client/transport/kafka.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0.
+from typing import Dict
+
+import attr
+
+from openlineage.client.run import RunEvent
+from openlineage.client.serde import Serde
+from openlineage.client.transport.transport import Transport, Config
+from openlineage.client.utils import get_only_specified_fields
+
+
+@attr.s
+class KafkaConfig(Config):
+    # Kafka producer config
+    # https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html#kafka-client-configuration
+    config: Dict[str, str] = attr.ib()
+
+    # Topic on which we should send messages
+    topic: str = attr.ib()
+
+    # Flush by default. The process that emits can be killed in some cases - for example
+    # in Airflow integration.
+    flush: bool = attr.ib(default=True)
```

Comment:
This is the opposite of what's in the README

Reply:
Clarified, both in readme and here.